test/TritonGPU/amd/amd-stream-loop-assume.mlir

```diff
@@ -0,0 +1,79 @@
+// RUN: triton-opt %s -split-input-file -tritonamdgpu-stream-pipeline="num_stages=2" -canonicalize | FileCheck %s --check-prefixes=LOCAL_1
```

Comment:
Nit: I'd think we want to just drop the customized `--check-prefix` to just use the default `CHECK` here given `LOCAL_1` doesn't actually reflect anymore given we are not adding the local-prefetch parameter.